### PR TITLE
fix: add updatePolicies

### PR DESCRIPTION
**Pull request by Gabriel-403**
add updatePolicies

### CODE REVIEW

File: src/model/model.ts [hunk 1]
```diff
@@ -306,6 +306,29 @@ export class Model {
...
+    if (newRules.length !== oldRules.length) {
+      throw new Error('The new policy does not match the old policy');
+    }
+    const rules = [];
+    for (let i = 0; i <= oldRules.length; i++) {
+      if (this.updatePolicy(sec, ptype, oldRules[i], newRules[i]) === true) {
+        rules.push(newRules[i]);
+      }
+    }
+    return [true, rules];
```
Comment (Zxilly) [inline src/model/model.ts:329]: Should only return a boolean.

File: src/persist/updatableAdapter.ts [hunk 2]
```diff
@@ -19,4 +19,5 @@ export interface UpdatableAdapter extends Adapter {
   // UpdatePolicy updates a policy rule from storage.
   // This is part of the Auto-Save feature.
   updatePolicy(sec: string, ptype: string, oldRule: string[], newRule: string[]): Promise<void>;
+  updatePolicies(sec: string, ptype: string, oldRule: string[][], newRule: string[][]): Promise<void>;
```
Comment (Zxilly) [inline src/persist/updatableAdapter.ts:22]: plz place this in batchAdapter Interface

File: src/util/util.ts [hunk 3]
```diff
@@ -18,7 +18,10 @@
 import { RoleManager } from '../rbac';
 
 function escapeAssertion(s: string): string {
-  return s.replace(/([rp])\.|[0-9]\./g, (match) => {
+  if (s.startsWith('r') || s.startsWith('p')) {
```
Comment (Zxilly) [inline src/util/util.ts:21]: You should make another PR for these.

**Review by Zxilly — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@Gabriel-403

**Comment by hsluoyz** [comment]
@Gabriel-403 why close?

**Pull request closed without merging**